src/java/test/org/apache/zookeeper/test/LoadFromLogTest.java

```diff
@@ -70,24 +52,37 @@
     private static final int TOTAL_TRANSACTIONS = NUM_MESSAGES + TRANSACTION_OVERHEAD;
     private volatile boolean connected;
 
-    /**
-     * test that all transactions from the Log are loaded, and only once
-     * @throws Exception an exception might be thrown here
-     */
-    @Test
-    public void testLoad() throws Exception {
+    private ZooKeeper zk;
+    private ServerCnxnFactory f;
```

Comment:
i know you didn't pick this variable name but I think it wouldn't be the worst thing to slip a change to this under the radar.